lib/buildsystems/cmake.rb

```diff
@@ -5,16 +5,16 @@ class CMake < Package
 
   attr_accessor :cmake_build_relative_dir
```

Comment:
Why is this not a property?

Reply:
Good call. Should probably only be a property...